Dockerfile

```diff
@@ -18,17 +18,12 @@ COPY pkg ./pkg
 RUN make build
 
 FROM registry.svc.ci.openshift.org/ocp/4.6:cli as origincli
-
 FROM registry.svc.ci.openshift.org/ocp/4.6:base
-RUN INSTALL_PKGS=" \
```

Comment:
do we know the base image includes dnf vs yum?  also minor point but this `rpm -V $INSTALL_PKGS` ensures the package actually installs and that we don't get false success.  If there are recommendations to an alternative please add

Reply:
I've backed out my changes to the original Dockerfile, you and Peri spotted too many flaws!

yum verifies packages after download (and fails if there's a problem) so the rpm -V is not necessary but not a big deal.

Reply:
This was not to verify the packages but to verify that every package I requested was actually found and installed.  I have seen builds complete the `yum install` but actually the run of the image failed because yum didnt find a package and therefore did not install it.  This was the point of this addition.  If yum has some other mechanism to verify that would be preferred IMO